rosdep/python.yaml

```diff
@@ -3465,6 +3469,10 @@ python3-pep8:
   ubuntu: [python3-pep8]
 python3-setuptools:
   ubuntu: [python3-setuptools]
+wheel:
```

Comment:
Wheel is available from native packages we prefer those than pip versions. 

https://packages.ubuntu.com/xenial/python-wheel
https://packages.debian.org/stretch/python-wheel
https://admin.fedoraproject.org/pkgdb/package/rpms/python-wheel/
https://packages.gentoo.org/packages/dev-python/wheel